src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractCreatePdpFeedServiceImpl.java

```diff
@@ -1,45 +1,43 @@
 package edu.cornell.kfs.concur.batch.service.impl;
 
+import java.io.File;
 import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
 import java.util.List;
 
+import org.kuali.kfs.sys.service.FileStorageService;
+
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractFile;
 import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractCreatePdpFeedService;
-import edu.cornell.kfs.concur.service.ConcurRequestExtractFileService;
-import edu.cornell.kfs.concur.service.ConcurRequestExtractFileValidationService;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractFileService;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractFileValidationService;
 
 public class ConcurRequestExtractCreatePdpFeedServiceImpl implements ConcurRequestExtractCreatePdpFeedService {
     private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractCreatePdpFeedServiceImpl.class);
+    protected FileStorageService fileStorageService;
     protected ConcurRequestExtractFileService concurRequestExtractFileService;
     protected ConcurRequestExtractFileValidationService concurRequestExtractFileValidationService;
```

Comment:
looks like the validation service is not used in this class. it should be removed.